eslzArm/managementGroupTemplates/policyDefinitions/policies.json

```diff
@@ -332,7 +357,7 @@
                                                     {
                                                         "type": "Microsoft.Consumption/budgets",
                                                         "apiVersion": "2019-10-01",
-                                                        "name": "default-sandbox-budget",
+                                                        "name": "[[parameters('effect')]",
```

Comment:
Is it intentional that the policy effect should be the resource name of the consumption/budgets resource name?

Reply:
Good spot, seems my last commit I forgot to push as I spotted this too 👍

Just pushed